index.html

```diff
@@ -1,28 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      rel="stylesheet"
-      href="bootstrap-5.2.0-beta1-dist/css/bootstrap.min.css" />
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Mulish:wght@200;700&display=swap"
-      rel="stylesheet"
-    />
-    <title>Trafalgar Landing Page</title>
-  </head>
-  <body class="mx-5">
-    <header></header>
-    <app-test></app-test>
-    <script src="https://code.jquery.com/jquery-2.1.3.min.js"></script>
-    <script src="bootstrap-5.2.0-beta1-dist/js/bootstrap.min.js"></script>
 
-    <!-- Web Components Import -->
-
-    <script src="web-components/test/test.js"></script>
-  </body>
-</html>
\ No newline at end of file
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Mulish:wght@200;700&display=swap" rel="stylesheet">
```

Comment:
3 font links seams to be to much. We have to clarify which one are used, and delete the rest.